tests/addons/core/test_botfront_testing_channel.py

```diff
@@ -0,0 +1,13 @@
+
+import pytest
+
+
+# this is needed so that the tests included as code examples look better
+from tests.addons.core.conftest import botfront_test_channel_test_data
```

Comment:
`conftest` files don't need to be imported, that's the whole point! (but you need to decorate what you want for them to become fixtures)

But... I don' think you need fixtures for that, since your here file is so short. Just put your data here.

Also, please split this spec in multiple tests (you have only one), and name them by the unuused "description" field you have in your data file. That way we'll see them appear in when running the spec.

Reply:
I've changed this to use the pytest.mark.parametrize rather than splitting it into multiple tests. hopefully this reaches a nice middle group between code repetition and ease of identifying failed tests.